apps/CMakeLists.txt

```diff
@@ -76,6 +76,9 @@ target_link_libraries(rt_swap_landmarks
     Boost::program_options
 )
 
+#add_executable(Image_Segmentation_2_0 ImageSegmentationMain.cpp SepDisegni.cpp)
+#target_link_libraries(Image_Segmentation_2_0 opencv_core opencv_imgcodecs opencv_highgui)
```

Comment:
Uncomment and get compiling
Executable name: `rt_segment_disegni`
Rename CPP to `SegmentDisegni.cpp`. 
Remove `SepDisegni` from the add executable command. 
Link against `rt::rtlib`